src/js/model/timestamp/timestamp.js

```diff
@@ -0,0 +1,40 @@
+var AMPParse = AMPParse || {};
+
+AMPParse.buildTimestamp = function (hexadecimal)
+{
+	// Defensive programming
+    if (typeof hexadecimal === "undefined" || typeof hexadecimal !== "string")
+    {
+	throw new ReferenceError("Provided parameter does not exist or is not a string");
+    }
+
+    //SDNV PARSER GOES HERE
+    sdnvReturn = AMPParse.buildSdnv(hexadecimal);
+    
+    // Snag the relevant bytes and build return value object
+    var ampEpoch = new Date(0);
+    ampEpoch.setUTCSeconds(1347148800);
+
+    var returnData = new Date(0);
+    returnData.setUTCSeconds(sdnvReturn.returnValue.value);
+
+    relative = false;
+    if (returnData < ampEpoch) {
```

Comment:
See sample code. To compare times you have to use `getTime()` off of the date object.